app/views/shared/_product_info_page.html.erb

```diff
@@ -10,9 +10,9 @@
 <% end %>
 <!-- small box before toggling -->
 <div class="toggling2">
-  <div class="row">
+  <div class="row adjust">
```

Comment:
Same as my comment in '_review_card.scss' above. You might want to use bootstrap spacing instead.